### DISCUSSION TITLE
Any known issues with Datapacks?

### DISCUSSION THREAD

**Discussion by Orholt** [EssentialsX/Essentials#4449] [Q&A] [answered]
Hi, I have 3 datapacks installed on my Paper 1.17.1-R0.1-SNAPSHOT server:
- Ships v12
- Illager Fortresses
- William Wythers Overauled Overwold

And only one working fine (William Wythers Overauled Overwold), for rest I'm just getting command blocks placed instead of structures:

[image](https://i.imgur.com/6e1KD0t.png)

I remember that this worked fine on my old Bukkit server (1.16) without Essentials installed

Any ideas of known issue connection bettwen Essentials and Datapacks?

**Comment by benwoo1110** [comment, marked as answer]
Doesn't really have any info to show essentials is causing it, how did you conclude that essentials is a potential issue? Have you tested without essentials installed or smt?

**Reply by Orholt** [reply]
I will after server update but now I'm guessing this might be problem with replacement of orginal minecraft commands with aliases
